server/database/config/build.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
+
+    DROP TABLE IF EXISTS users, events, news_tags, topics,event_topic, event_categories, primary_tags, secondary_tags, news CASCADE;
+
+    CREATE TABLE users (id SERIAL PRIMARY KEY, first_name TEXT NOT NULL, last_name TEXT NOT NULL, email UNIQUE TEXT NOT NULL, password TEXT NOT NULL, avatar TEXT, business_type TEXT NOT NULL, website TEXT NOT NULL, org_name TEXT NOT NULL, address TEXT NOT NULL, city TEXT NOT NULL, country TEXT NOT NULL, postal_code TEXT NOT NULL, facebook TEXT NOT NULL, instagram TEXT NOT NULL, twitter TEXT NOT NULL);
```

Comment:
table name is singular. "user", event, public-service, public-service_secondary_tag, topic, event_topic, primary_tag, secondary_tag, event_category,

Reply:
change org_name to organization

Reply:
but shouldn't the table name be plural?

Reply:
Has to be single